add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
```

Comment:
The database connection is being included directly in the script, which may not be optimal for larger applications. Consider using a Service or Repository pattern to handle database operations, which will improve maintainability and testability.

Create a `BookRepository` class:

```php
class BookRepository {
    private $connection;

    public function __construct($connection) {
        $this->connection = $connection;
    }

    public function addBook($title, $author, $year) {
        $stmt = $this->connection->prepare("INSERT INTO books (title, author, year) VALUES (?, ?, ?)");
        $stmt->bind_param("ssi", $title, $author, $year);
        return $stmt->execute();
    }
}
```

Then use this class in your script:

```php
$bookRepository = new BookRepository($connection);
if ($bookRepository->addBook($title, $author, $year)) {
    echo "Book added successfully!";
} else {
    echo "Error: " . $connection->error;
}
```